File: matlab_2_r.R
install.packages("R.matlab")
subjects <- list()
subs <- c("661","662","663","664","665","666","667","668","669","670")

for(sub in subs){
temp_name=paste0("s", sub)
temp_data <- readMat(paste0("~/Box Sync/data_drop/",sub,"/R_data.mat"))
unclass(temp_data)
temp_data$weak.strong <- as.numeric(temp_data$weak.strong[sapply(temp_data$weak.strong,function(x) x=="1"|x=="4")])
subjects[[temp_name]] <- temp_data
}

File: matlab_2_r.m
subs={'661','662','663','664','665','666','667','668','669','670'};

for sub=1:length(subs)
cd(['~/Box Sync/data_drop/' subs{sub}]) 
v=dir('*test.mat');
load(v.name)
b=[theData.image_test];
names=fieldnames(b);
for n=1:length(names)
c.(names{n})={b.(names{n})};
c.(names{n})=cat(1,c.(names{n}){:});
end
save('R_data.mat','-struct','c')
end